src/app/clusters/ethernet-network-diagnostics-server/ethernet-network-diagnostics-server.cpp

```diff
@@ -76,6 +76,8 @@ CHIP_ERROR EthernetDiagosticsAttrAccess::ReadPHYRate(AttributeValueEncoder & aEn
     Attributes::PHYRate::TypeInfo::Type pHYRate;
     PHYRateType value = EmberAfPHYRateType::EMBER_ZCL_PHY_RATE_TYPE_10_M;
 
+    pHYRate.SetNull();
```

Comment:
Is this change intentional?

Reply:
Yes, CI was failing esp32 due to non initialized

Reply:
@tcarmelveilleux Which CI was failing?  I thought we had made that warning not fail in all the cases that matter....

In particular, `pHYRate`'s constructor already did the equivalent of `SetNull`.....

Reply:
It's because `Attributes::PHYRate::TypeInfo::Type pHYRate;` is never assigned a value. Somehow the compiler used for ESP32 tests blew-up on that. Let me take it out and find the run again.